syncTextures.js

```diff
@@ -32,6 +32,25 @@ const textureFolder = './textures';
 // delay (milliseconds) between file download requests
 const delay = '10';
 
+/**
+ * Warns the user if they are running an older version of NodeJS
+ * Only checks major version
+ *
+ */
+function checkNodeVersion() {
+  const minVersion = '19.0.1';
+  const installed = process.versions.node;
+
+  if (installed.split('.'[0]) < minVersion.split('.'[0])) {
+    console.log(
+      `\n WARNING! You are running an untested version of NodeJS!
+      Installed: ${installed}
+      Tested: ${minVersion}
+      Should any errors arise, try updating Node and NPM to the latest tested versions before reporting.\n`
```

Comment:
This is a little confusing, since the version they are running was probably tested by someone. I suggest that you say simply, "Warning: This script was verified to work with version {minVersion} of Node and NMP and you are on earlier versions. You can update both Node and NMP with the command: npm install @latest"  (but verify that this is the right command)